src/index.js

```diff
@@ -11,28 +10,25 @@ const opn = require('opn');
 
 const schema = require('./schema');
 const config = require('./config/config');
-
 const syncRouter = require('./route/sync');
 const apiRouter = require('./route/api');
-
 const startSync = require('./sync');
 
 const PORT = 5555;
 
+// Create log dir if needed
+const dir = config.LOG_PATH;
+if (!fs.existsSync(dir)){
```

Comment:
nit: existence check can be moved to logger module exports too

Reply:
done, moved to logger.js